core/templates/dev/head/i18n.js

```diff
@@ -39,7 +39,9 @@ oppia.constant('DEFAULT_TRANSLATIONS', {
 
 oppia.controller('I18nFooter', [
   '$http', '$rootScope', '$scope', '$translate', '$timeout', '$cookies',
-  function($http, $rootScope, $scope, $translate, $timeout, $cookies) {
+  'UserService',
+  function(
+      $http, $rootScope, $scope, $translate, $timeout, $cookies, UserService) {
```

Comment:
Please have the line breaks in this list match those in the list above.

/cc @apb7 -- this should be a lint check.

Reply:
Filed #5809. Thanks, Sean!

Reply:
Done.